static/src/stylesheets/module/facia/_slices.scss

```diff
@@ -171,8 +171,8 @@ Hence why a greater depth of selector specificity is needed.
         display: none;
     }
 
-    .fc-item__live-indicator .live-pulse-icon {
-        display: none;
+    .live-pulse-icon:before {
+        content: none;
```

Comment:
just a question, what's you're opinion on keeping all styles for a selector in the same place? e.g, moving this into `_live-blog.scss`, inside `.live-pulse-icon:before`?

```
...
    @include animation(live-pulse 1s infinite);

    .fc-slice--popular & {
        content: none;
    }
}
```